.devcontainer/default/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+// Reference: https://containers.dev/implementors/json_reference/
+{
+  "name": "SourceKit LSP",
+  "dockerFile": "Dockerfile",
+
+  // Allow the processes in the container to attach a debugger
+  "capAdd": [ "SYS_PTRACE" ],
+  "securityOpt": [ "seccomp=unconfined" ],
+
+  "mounts": [
+    // Use a named volume for the build products for optimal performance (https://code.visualstudio.com/remote/advancedcontainers/improve-performance?WT.mc_id=javascript-14373-yolasors#_use-a-targeted-named-volume)
+    "source=${localWorkspaceFolderBasename}-build,target=${containerWorkspaceFolder}/build,type=volume"
+  ],
+  "remoteEnv": {
+    // `/usr/bin` must be first: https://github.com/apple/sourcekit-lsp/blob/main/Documentation/Development.md#linux
+    "PATH": "/usr/bin:${containerWorkspaceFolder}/.build/bin:${containerEnv:PATH}" },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "sswg.swift-lang",
+        "vknabel.vscode-apple-swift-format"
```

Comment:
Similar to my comment above, swift-format should be run using the *Run swift-format* action, so this shouldn’t be needed anymore.

Reply:
This extension uses `swift-format` from the project if it is a dependency. I like including it because it makes other VSCode functionality (like format-on-save) work correctly.

Reply:
You can get format on save to work using SourceKit-LSP in Swift 6.0. The vscode extension shouldn't be needed anymore.

Reply:
You are correct! Sorry I didn't realize this was added.